moped-editor/src/views/projects/projectView/ProjectTeam/ProjectTeamTable.js

```diff
@@ -103,8 +118,16 @@ const useColumns = ({
       {
         headerName: "Workgroup",
         field: "moped_workgroup",
+        editable: true,
         width: 200,
-        valueGetter: (workgroup) => workgroup?.workgroup_name,
```

Comment:
valueGetter changes the value that is sent to the edit cell, valueFormatter will just format the value for the non-edit render.